prod/README.md

```diff
@@ -9,27 +9,35 @@ Container includes installation of:
 - corsika and sim\_telarray
 - miniconda
 - packages required by simtools (from environment.yml)
-- simtools (master)
+- simtools (main branch)
 
-Images are automatically built by the [Github action workflow ../.github/workflows/build-image.yml](../.github/workflows/build-image.yml) and can be downloaded from the [gammasim package website](https://github.com/orgs/gammasim/packages).
+Images are automatically built by the [Github action workflow ../.github/workflows/build-image.yml](../.github/workflows/build-image.yml) and can be downloaded from the [gammasim package website](https://github.com/gammasim/containers/pkgs/container/simtools-prod).
 
 ## Running
 
 To run the container in bash 
-
 ```
-docker run --rm -it -v "$(pwd)/external:/workdir/external" ghcr.io/gammasim/simtools-prod:latest bash
+docker run ghcr.io/gammasim/simtools-prod:latest bash
 ```
+In the container, simtools applications are installed and can be called directly (e.g., `simtools-prod simtools-print-array-elements -h`).
 
-In the container, find the simtools directory in `/workdir/simtools/`.
+In case file exchange with the local file system is required, use the docker syntax to mount a directory. Example:
+```
+docker run --rm -it -v "$(pwd):/workdir/external" ghcr.io/gammasim/simtools-prod:latest bash
+```
 
-To run an application inside the container, e.g.:
+The following examples runs an application inside the container and write the output into a directory of the local files system, 
```

Comment:
This sentence should be in singular I think cause there's only one example below.